app/cdap/components/WranglerV2/SearchResultsList/SearchResultsList.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+import SearchResultList from 'components/WranglerV2/SearchResultsList';
+
+export default {
+  title: 'SearchResultList',
+  component: SearchResultList,
+};
+
+const handleClick = (data) => action('clicked')(data);
+
+export function Default(args) {
+  return <SearchResultList {...args} />;
+}
+
+Default.args = {
+  resultsList: [
```

Comment:
can we reuse the same resultsList as in other component storybook
if yes, export from there and import here, don't move it out of file

Reply:
created a variable and exports are not possible as the storybook considers all exports as stories